seatunnel-core/seatunnel-spark-starter/pom.xml

```diff
@@ -74,7 +74,7 @@
 
         <dependency>
             <groupId>org.apache.seatunnel</groupId>
-            <artifactId>seatunnel-translation-spark-2.4</artifactId>
+            <artifactId>seatunnel-translation-spark-common</artifactId>
```

Comment:
How user switch seatunnel-transform-spark(2.4、3.3) list?

Reply:
How to choose? Seem like you should add jar choose logic in seatunnel-spark-starter.

Reply:
I prefer a more automated way, such as by judging the version of SPARK_HOME or using SPARK2 by default, you can switch versions by adding parameters to the shell script

Reply:
Ok for me